StratisCore.UI/src/app/wallet/staking/staking.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container>
-  <div *ngIf="!wallet || wallet.hasBalance">
+  <div *ngIf="wallet?.hasBalance">
```

Comment:
So interesting,  wallet is never actually set on the  staking component. It is always null so the Staking Component will never show with this code.
need this on line 51 of Dashboard Component also
``` <app-staking [wallet]="(wallet| async)"></app-staking>```